Adding documentation to use the FIPS sidecar for GOVCLOUD support (#17874)
DataDog/documentation: content/en/containers/amazon_ecs/_index.md

Instead, run the Agent in bridge mode with port mapping to allow easier retrieval of [host IP through the metadata server][6].

> #### FIPS Proxy for GOVCLOUD environments.

Reviewer: I recommend you add the site region shortcode to this PR, so that the new content only appears when the US1-FED site region is selected from the Datadog Site drop-down menu.

Author: done - Thanks for the rec.

> **This feature is available for Linux only**
> 
> If you want to send data to Datadog's GOVCLOUD datacenter, you will need to add the `fips-proxy` sidecar container and open container ports to ensure proper communication for all the features:

Reviewer: ```suggestion
To send data to Datadog's GOVCLOUD datacenter, add the `fips-proxy` sidecar container and open container ports to ensure proper communication for all of the features. 

**Note**: This feature is available for Linux only.
```

                    "protocol": "tcp"
                },
                {
                    "containerPort": 9809,
                    "protocol": "tcp"
                },
                {
                    "containerPort": 9810,
                    "protocol": "tcp"
                },
                {
                    "containerPort": 9811,
                    "protocol": "tcp"
                },
                {
                    "containerPort": 9812,
                    "protocol": "tcp"
                },
                {
                    "containerPort": 9813,
                    "protocol": "tcp"
                },
                {
                    "containerPort": 9814,
                    "protocol": "tcp"
                }
            ],
            "essential": true,
            "environment": [
                {
                    "name": "DD_FIPS_PORT_RANGE_START",
                    "value": "9803"
                },
                {
                    "name": "DD_FIPS_LOCAL_ADDRESS",
                    "value": "127.0.0.1"
                }
            ]
        }
   ],
   "family": "datadog-agent-task"
}
```

> Secondly, you will need to update the environment variables of the Datadog Agent's container to enable sending traffic through the FIPS proxy:

Reviewer: ```suggestion
You also need to update the environment variables of the Datadog Agent's container to enable sending traffic through the FIPS proxy:
```
